src/bindActionCreators.js

```diff
@@ -1,5 +1,7 @@
 function bindActionCreator(actionCreator, dispatch) {
-  return (...args) => dispatch(actionCreator(...args))
+  const boundActionCreator = (...args) => dispatch(actionCreator(...args))
+  boundActionCreator.__boundContext__ = dispatch
```

Comment:
Originally, I was checking to see if the `__boundContext__ === dispatch` but I think I left out the right-hand side of that check while moving things around.

I've updated it to use `__isBoundActionCreator__` instead, because `__boundContext__` was a misnomer, since there isn't actually a bound context on the function via `Function.prototype.bind`.